lib/resources/sample.js

```diff
@@ -0,0 +1,77 @@
+const crypto = require('crypto')
+const got = require('got')
```

Comment:
```suggestion
const got = require('got').default
```

permit DX suggestions